vaadin-crud-flow-parent/vaadin-crud-flow/src/test/java/com/vaadin/flow/component/crud/CrudTest.java

```diff
@@ -93,6 +94,12 @@ public void getToolbarVisible_setVisibleToFalse_returnsFalse() {
         Assert.assertEquals(false, systemUnderTest.getToolbarVisible());
     }
 
+    @Test
+    public void crudHasStyle() {
+        Crud<String> c = new Crud<>();
+        Assert.assertTrue(c instanceof HasStyle);
```

Comment:
Let's reuse an already existing instance, no need to create a new one.

```suggestion
        Assert.assertTrue(systemUnderTest instanceof HasStyle);
```